instrumentation/java-http-client/library/src/main/java/io/opentelemetry/instrumentation/httpclient/internal/Experimental.java

```diff
@@ -14,6 +14,7 @@
  * APIs (or a version of them) may be promoted to the public stable API in the future, but no
  * guarantees are made.
  */
+@Deprecated
```

Comment:
Is it necessary to add javadoc that tells users to use `io.opentelemetry.instrumentation.javahttpclient.internal. Experimental` instead here?